### DISCUSSION TITLE
[BUG] Getting "Error: Unsupported Server Component type: `undefined`" with `<IconLogin2 />`

### DISCUSSION THREAD

**Discussion by hexcowboy** [tabler/tabler#1767] [Q&A] [answered]
### Browser

Firefox 119.0

### OS

macOS Ventura

### Screen size

1920x1080

### Describe the bug

With Next.js, I'm getting this stack trace.

![image](https://github.com/tabler/tabler/assets/8162609/7bb4ca92-4741-4625-9c82-d119af303ef5)

I'm able to reproduce by using the `<IconLogin2 />` component, and I haven't found any other components that cause this error.

### How to reproduce

1. Start a new Next.js app
2. Import and use the `<IconLogin2 />` component

**Comment by 00gigs** [comment]
just had the same issue and it was an overlooked mistake on my end.i forgot to capitalize my react component 

I had to change the following from

const firebaseConfig =() =>{

}
export default FirebaseConfig

to

const FirebaseConfig =() =>{

}
export default FirebaseConfig.

**Reply by 00gigs** [reply]
hope this helps

**Reply by 00gigs** [reply]
I also removed curly brackets in

 import {FirebaseCrud} from '../app/components/Firebasecrud/firebasecurd'
 
 to 
 
  import FirebaseCrud from '../app/components/Firebasecrud/firebasecurd'
  
  had to do both changes to get resolve error

**Comment by hexcowboy** [comment, marked as answer]
This issue was fixed in a previous release
